rules/windows/persistence_local_scheduled_task_scripting.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/11/29"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/29"
+
+[rule]
+author = ["Elastic"]
+description = "A scheduled task was created by a Windows script via cscript.exe, wscript.exe or powershell.exe, this can be used by an adversary to establish persistence."
+false_positives = ["Legitimate scheduled tasks may be created during installation of new software."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Scheduled Task Created by a Windows Script"
+note = "Decode the base 64 encoded Tasks Actions registry value to investigate the task configured action."
+risk_score = 43
+rule_id = "689b9d57-e4d5-4357-ad17-9c334609d79a"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan = 5s
```

Comment:
I noticed the `maxspan` was set to 30s in your screenshot.  Is 5 seconds too short to connect the events? Should we up it a little more?

Reply:
works also with 5s, but agree to be in the safe side I'll set it to 30s